assets/js/components/CompleteModuleActivationCTA.js

```diff
@@ -40,43 +40,42 @@ const CompleteModuleActivationCTA = ( { slug, title, description } ) => {
 	const module = useSelect( ( select ) => select( MODULES_STORE ).getModule( slug ) );
 	const adminReauthURL = useSelect( ( select ) => select( `modules/${ slug }` )?.getAdminReauthURL() );
 	const canManageOptions = useSelect( ( select ) => select( CORE_USER ).hasCapability( PERMISSION_MANAGE_OPTIONS ) );
-	const { name } = module;
 
 	const onCTAClick = useCallback( async () => {
-		global.location = adminReauthURL;
-	} );
+		global.location.assign( adminReauthURL );
+	}, [ adminReauthURL ] );
```

Comment:
* Used `location.assign` instead
* Added the selected `adminReauthURL` to the callback's dependencies – no array was present before which would cause the callback to be updated each time the component was rendered. Probably not an expensive omission but if we're using `useCallback` we should always provide a dependency list, otherwise it defeats the purpose.